CHANGELOG.md

```diff
@@ -8,6 +8,8 @@
 * Using in-house `GenericDataChunkIterator` [PR #1068](https://github.com/catalystneuro/neuroconv/pull/1068)
 
 ## Improvements
+* Do not run dev tests on PRs [PR #1092](https://github.com/catalystneuro/neuroconv/pull/1082)
```

Comment:
1092 is showing up twice, also the link is wrong

Reply:
Fixed